### DISCUSSION TITLE
Website down

### DISCUSSION THREAD

**Discussion by AIR887** [community/community#64568] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi there,

I use GitHub to run my website and have been working on it so far and everything has been looking great and working. Suddenly I go and try to update my website, and when I open it, nothing comes up and only the domain comes up.
<img width="927" alt="Screenshot 2023-08-21 223205" src="https://github.com/community/community/assets/90001365/2a3fe272-5d5b-4b4a-90d1-27152d356c1e">

Just a few days ago it was working perfectly fine and running well.
Please help me as i need it to be fixed as soon as possible.

**Comment by anztim** [comment]
I checked your DNS records:

```
;; QUESTION SECTION:
;mctab.ca.                      IN      ANY

;; ANSWER SECTION:
mctab.ca.               360     IN      A       198.54.114.202
mctab.ca.               85560   IN      NS      dns1.namecheaphosting.com.
mctab.ca.               85560   IN      NS      dns2.namecheaphosting.com.
```

You should remove the existing A record and add a new CNAME record point to your original GitHub page address.

**Comment by geoffreylgv** [comment, marked as answer]
Hello @AIR887, Here are some steps to bring your website to live

**Recommendation**: _Remove the A record you have, but you can follow a long to edit it and good to go_

0. Go to your repository (yours here is **MyWebsite**)
1. Go to the Settings>Pages
2. Add your custom domain name `mctab.ca` and click on save
3. Go to namecheap.com, sign in and go to **Dashboard**
4. On the dashboard, find your domain, click on **Manage** 
5. Then click on **Advanced DNS**
6. Click on **Add new record** , Select **A Record** add x3 and **CNAME Record** x1
See the table below to perform the step 6

|Type|Host|Value|TTL|
|--|--|--|--|
|A record|@|185.199.108.153|Automatic|
|A record|@|185.199.109.153|Automatic|
|A record|@|185.199.110.153|Automatic|
|A record|@|185.199.111.153|Automatic|
|CNAME Record|www|_username_.github.io|60min|

_Replace "username" by your GitHub username._

7. come back to your repo settings and check the step 2.  You'll see your page is live and available showing up what you coded

For more information take a look at
[Configuring DNS : An-apex-domain step 5](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain)

**Reply by AIR887** [reply]
Hey there, 
Thank you for your response. When I go to advanced DNS, it doesn't let me add a new record.
<img width="605" alt="image" src="https://github.com/community/community/assets/90001365/4e4c6ecc-51b7-40f2-a0ab-c834892f233b">

I remember I had the same issue from before and somehow fixed it, but now I cant remember how to now. 
I would really appreciate if you can help with this.
Thanks a lot!

**Reply by geoffreylgv** [reply]
Hey hi @AIR887 ; time zone, sorry for delay
Ok, we can have a meeting for that. 
Let me know

**Reply by AIR887** [reply]
Thanks a lot for your time, i have found a solution regarding my problem and the website is back up

Thanks a lot!

**Reply by geoffreylgv** [reply]
Awesome, that's nice. 
If you don't mind, can you share the update with us, like what was the key resolver beside the usual configuration of the DNS ?
Thank you too.

**Reply by AIR887** [reply]
I went to the cPanel on namecheap and went to a place called Zone Editor that would let me add records, but it was a harder process and took longer to do than the normal way that you do it, and since I didn't have the easier option way for some reason, I had to do this way.
However it worked just like the way you told me to do it except not through the Advanced DNS way that you reccomeneded. 

Hope this answers your question, if not please let me know and I will try and rephrase what I said

**Reply by geoffreylgv** [reply]
Alright, that's very helpful to know.

And yes, it answered my question. 
I hope this helps someone else who encounters the same problem.

Thank you.

**Comment by cdxgh** [comment]
I understand your concern about your website suddenly not displaying correctly. There could be various reasons for this issue, and troubleshooting it would typically involve checking several aspects. Here's a step-by-step guide to help you diagnose and potentially resolve the problem:

1. **Check Your Domain Configuration:**
   - Ensure that your domain is correctly configured to point to your GitHub Pages repository. Sometimes, domain settings can get disrupted, leading to issues like this.
   - Double-check the DNS records and make sure they are correctly set to point to GitHub Pages servers.

2. **GitHub Repository:**
   - Verify that your GitHub repository, which hosts your website's source code, is still active and contains the latest version of your website.
   - Check if any recent changes or commits may have inadvertently affected the functionality of your site.

3. **GitHub Pages Settings:**
   - Go to your GitHub repository's "Settings" tab.
   - Scroll down to the "GitHub Pages" section and ensure that it's set to use the correct branch (usually `main` or `master`) and that the source directory is correctly specified.
   - Check if there are any error messages in this section that might indicate the problem.

4. **Check for Errors:**
   - Open your browser's developer tools (usually by pressing F12 or right-clicking and selecting "Inspect").
   - Go to the "Console" tab, and check for any JavaScript errors or network-related errors that might provide more insight into the issue.

5. **Cache Issues:**
   - Sometimes, your browser may cache old versions of your website. Try clearing your browser's cache and reloading the page.

6. **GitHub Status:**
   - Check the GitHub Status page (https://www.githubstatus.com/) to see if there are any ongoing issues or outages that might be affecting GitHub Pages.

7. **Contact GitHub Support:**
   - If the issue persists and you can't identify the problem, consider reaching out to GitHub Support for assistance. They can help with issues related to GitHub Pages and repository hosting.

8. **Backup and Restore:**
   - As a last resort, if you have a backup of your website, you can try restoring it to a working state. GitHub's version control system should help with this.

Remember to document any changes you make and keep an eye on error messages. If you're still unable to resolve the issue, contacting GitHub Support is a good option, as they can provide more specific guidance based on the details of your situation.

Please be cautious while making changes to your GitHub repository and domain settings to avoid any accidental data loss.
